internal/cli/runner_agent.go

```diff
@@ -125,12 +129,19 @@ func (c *RunnerAgentCommand) Run(args []string) int {
 				"Error starting liveness server: %s", err.Error(),
 				terminal.WithErrorStyle(),
 			)
+			cancel()
```

Comment:
Doesn't the `return 1` below exit us already? So we're canceling something that'll never be run, aren't we?

Reply:
I wasn't 100% sure on if the `cancel` was needed here, but I imagine you're correct and it's not needed. I removed it in 74e7982a